halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/EchoProfileFactory.java

```diff
@@ -154,12 +152,13 @@ private String halyardVersion() {
     }
   }
 
+  /** Translates Halyard's `Stats` class to Echo's `Telemetry` class */
   @Data
   private static class TelemetryWrapper {
-    private Telemetry telemetry;
+    private Stats telemetry;
 
-    TelemetryWrapper(Telemetry telemetry) {
-      this.telemetry = telemetry;
+    TelemetryWrapper(Stats stats) {
+      this.telemetry = stats;
     }
   }
```

Comment:
Can you _also_ write it out to `stats` and then update echo to read it from that location instead?

Reply:
Good point, I had missed that in my review of the PR.  In general, I'd like to minimize the difference between what is in the hal config and what is in each service's config, both to simplify the logic in `kleat` and also to make it easier on anyone trying to understand how these settings map to each other.

Reply:
Opened up two other PRs in echo and buildtool to make this happen.